v3/src/components/codap-component.scss

```diff
@@ -9,4 +9,18 @@
       color: black !important
     }
   }
+
+  .component-resize-handle {
+    position: absolute;
+    bottom: 0px;
+    right: 0px;
+    width: 22px;
+    height: 22px;
+    z-index: 999;
```

Comment:
Does it need to be that high? We don't want it in front of dialogs or inspectors.